src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.HttpContentAsyncStream.cs

```diff
@@ -142,6 +142,15 @@ public override Task<int> ReadAsync(byte[] buffer, int offset, int count, Cancel
                 }
             }
 
+            public override void Write(byte[] buffer, int offset, int count)
+            {
+                // WriteAsync should be how writes are performed on this stream as
+                // part of HttpContent.CopyToAsync.  However, we implement Write just
```

Comment:
Thanks, but I'd like to leave this as is.  The comment doesn't state anything about overriding CopyToAsync, but rather states that Write happens as part of CopyToAsync, which includes anything it invokes, and what it invokes is an implementation detail from the perspective of CurlHandler.  Since CurlHandler is simply invoking CopyToAsync, the comment is calling out that these writes happen as part of that.